minard/views.py

```diff
@@ -104,6 +104,10 @@ def state(run=None):
     if run_state['caen'] is not None:
         caen_state = detector_state.get_caen_state(run_state['caen'])
 
+    #tubii_state = None
+    #if run_state['tubii'] is not None:
```

Comment:
@Eric, if I comment out these lines it gives a TUBii translation error in detector_state.py. So its still not quite there. Any idea what I'm doing wrong?